WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -26,6 +27,8 @@
 
 import javax.inject.Inject;
 
+import de.greenrobot.event.EventBus;
```

Comment:
Me neither, would have thought of using a LocalBroadcastReceiver to keep it more Androidish but no worries, we're using this mechanism elsewhere in the app and I agree it's the simplest of the alternatives when both things happen in the same process